packages/office-ui-fabric-react/src/components/FloatingPicker/PeoplePicker/FloatingPeoplePicker.tsx

```diff
@@ -12,25 +12,20 @@ export class BaseFloatingPeoplePicker extends BaseFloatingPicker<IPersonaProps,
 
 export class FloatingPeoplePicker extends BaseFloatingPeoplePicker {
   // tslint:disable-next-line:no-any
-  public static defaultProps: any = {
+  public static defaultProps: Partial<IBaseFloatingPickerProps<IPersonaProps>> = {
     onRenderSuggestionsItem: (props: IPersonaProps, itemProps?: IBasePickerSuggestionsProps) =>
       SuggestionItemNormal({ ...props }, { ...itemProps }),
-    createGenericItem: createItem
+    createForceResolvedItem: createItem
   };
 }
 
-export function createItem(name: string, isValid: boolean): ISuggestionModel<IPersonaProps> {
+export function createItem(name: string): ISuggestionModel<IPersonaProps> {
   // tslint:disable-next-line:no-any
   const personaToConvert: any = {
     key: name,
     primaryText: name,
-    imageInitials: '!',
-    isValid: isValid
```

Comment:
`key` and `isValid` do  not exist on IPersona. This component seems half written against ExtendedPersona and half written against IPersona.

Looking for feedback here on where to put these props. My first instinct is that 

- `key` is a valuable optional prop for pickable items (and maybe we should enforce that pickable items take an optional key),
- `isValid` should either be entirely the concern of the implementer (e.g. they provide their own custom persona types, see #8465 ), or should live on the ISuggestionModel, as it's not an intrinsic attribute that most implementers are going to have on their data model.